### PR TITLE
Configure dependabot

### PR DESCRIPTION
**Pull request by piotrmiskiewicz**
**Description**

Changes proposed in this pull request:
- configure dependabot

**Related issue(s)**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "gomod"
+    directory: "/"
+    labels:
+      - "go"
```
Comment (MarekMichali) [inline .github/dependabot.yml:8]: This label doesn't exist in this repo.

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      - "kind/enhancement"
+    schedule:
+      interval: "daily"
+    commit-message:
+      prefix: "gomod"
+      include: "scope"
+  - package-ecosystem: "docker"
+    directory: "/"
+    labels:
+      - "docker"
```
Comment (MarekMichali) [inline .github/dependabot.yml:20]: This label doesn't exist in this repo.

Comment (piotrmiskiewicz) [reply .github/dependabot.yml]: removed

File: .github/dependabot.yml (hunk 1 above)
Comment (piotrmiskiewicz) [reply .github/dependabot.yml]: removes

**Review by MarekMichali — APPROVED** [review]

**Pull request merged**